04/main.py

```diff
@@ -0,0 +1,33 @@
+"""
+Home Work 04
+Dmytro Verovkin
+robot_dreams
+"""
+
+
+def main():
+    user_input = input("Please enter text: ")
+
+    # check for integer
+    try:
+        if int(user_input) % 2 == 0:
+            print("You entered EVEN number")
+        else:
+            print("You entered ODD number")
+    except ValueError:
+
+        # check if it's float
+        try:
+            if float(user_input):
+                print("You entered a float number, it can't be ODD or EVEN")
+        except ValueError:
+
+            # check if it's a word
+            if user_input.isalpha():
```

Comment:
This check can be removed here because any text with a dot `.` symbol, for example, will be recognized as the mysterious one. At this point, we already know that this is not a digit, so we need just cont the length of it

Reply:
if the input be like '@#$#$' it wouldn't be a word

Reply:
Fare enough. According to the home task that should be a word. let's assume that there can be any text instead of work